rules/scala/workspace.bzl

```diff
@@ -21,12 +21,12 @@ def scala_artifacts():
         "org.scala-sbt:util-interface:1.3.0",
         "org.scala-sbt:util-logging_2.12:1.3.0",
         "org.scala-sbt:compiler-interface:1.3.4",
```

Comment:
Let's also version bump the other scala-sbt libraries. Other than the `test-interface` one, they're all versioned in line with Zinc

Reply:
Just did thid